tests/test_run_geofabrics/test_run_geofabrics.py

```diff
@@ -73,7 +73,7 @@ def setUpClass(cls):
         # Load in environment variables to get and set the private API keys
         dotenv.load_dotenv()
```

Comment:
Update the test to make the Linux test less strict. GeoFabrics is sensitive of numerical changes between operating systems. The benchmarks are created in Windows so allow some changes when testing in Linux. 

Make the thresholds larger.

Reply:
Made a new benchmark in linux